docs/dev.rst

```diff
@@ -428,107 +552,43 @@ this code and it has to be tested manually. You can do something like this:
 11. click on "Sign out"
 
 
-Database migrations
--------------------
-
-The Symbols Service webapp uses Django's ORM and thus we do database migrations
-using Django's migration system.
-
-Do this:
-
-.. code-block:: shell
-
-   $ make shell
-   app@xxx:/app$ ./manage.py makemigration --name "BUGID_desc" APP
-
-
-Accounts and first superuser
-----------------------------
-
-The Symbols Service webapp has an accounts system.
-
-Users need to create their own API tokens but before they can do that they need
-to be promoted to have that permission at all.
-
-The only person/people who can give other users permissions is the superuser.
-To bootstrap the user administration you need to create at least one superuser.
-That superuser can promote other users to superusers too.
-
-This action does NOT require that the user signs in at least once. If the user
-does not exist, it gets created.
-
-The easiest way to create your first superuser is to use ``docker compose``:
-
-.. code-block:: shell
-
-   $ docker compose run --rm web bash python manage.py superuser yourname@example.com
-
-Additionally, in a local development environment, you can create a
-corresponding user in the oidcprovider service like this:
-
-.. code-block:: shell
-
-   $ docker compose exec oidcprovider /code/manage.py createuser yourname yourpassword yourname@example.com
-
-
-Giving users permission to upload symbols
------------------------------------------
-
-The user should write up a bug. See :ref:`upload-basics`.
-
-If the user is a Mozilla employee, needinfo the user's manager and verify the
-user needs upload permission.
+System tests
+------------
 
-If the user is not a Mozilla employee, find someone to vouch for the user.
+System tests are located in the repository in ``systemtests/``. See the
+``README.rst`` there for usage.
 
-Once vouched:
+System tests can be run against any running environment:
 
-1. Log in to `<https://symbols.mozilla.org/users>`_
-2. Use the search filter at the bottom of the page to find the user
-3. Click to edit and make give them the "Uploaders" group (only).
-4. Respond and say that they now have permission and should be able to either
-   upload via the web or create an API Token with the "Upload Symbol Files"
-   permission.
-5. Resolve the bug.
+* local: local dev environment
+* stage: the stage server environment
+* prod: the prod server environment--will not run destructive tests
 
+System tests can help verify that upload API and download API work. They
+periodically need to be updated as symbols files expire out of the systems.
 
-Viewing all metrics keys
```

Comment:
This section is busted. The `LogAllMetricsKeys` thing is written for an older version of markus and needs to be updated. I wrote up bug 1867570 to rework this.